### PR TITLE
Add Google CA values

### PR DESCRIPTION
**Pull request by pitlv2109**
Similar to https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/example-values/values-istio-googleca.yaml.

### CODE REVIEW

**Review by howardjohn — COMMENTED** [review]
The example addition lgtm, but shouldn't add all the other file changes here. Not sure where they came from?

**Comment by pitlv2109** [comment]
> The example addition lgtm, but shouldn't add all the other file changes here. Not sure where they came from?

From running make gen and stuff like that. Should I have not run it?

**Comment by howardjohn** [comment]
@pitlv2109 you should have but they don't have anything to do with this change (eg indentation) which is a bit suspicious to me.. I am not familiar with the generation logic here though

**Comment by pitlv2109** [comment]
> @pitlv2109 ...I am not familiar with the generation logic here though

Same. I ran the typical `BUILD_WITH_CONTAINER=0 make update-goldens; make clean gen`. Maybe I have different dependency versions

File: cmd/mesh/testdata/operator/output/operator-init.yaml [hunk 1]
```diff
@@ -4,121 +4,121 @@ metadata:
...
-      - events
-      - namespaces
-      - pods
-      - persistentvolumeclaims
-      - secrets
-      - services
-      - serviceaccounts
-    verbs:
-      - '*'
+# istio groups
```
Comment (ostromart) [inline cmd/mesh/testdata/operator/output/operator-init.yaml]: The indentation change looks suspect to me. Maybe introduced by your editor?

File: cmd/mesh/testdata/operator/output/operator-init.yaml [hunk 2]
```diff
@@ -4,121 +4,121 @@ metadata:
...
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: istio-operator
 subjects:
-  - kind: ServiceAccount
-    name: istio-operator
-    namespace: istio-operator
+- kind: ServiceAccount
+  name: istio-operator
+  namespace: operator-test-namespace
```
Comment (ostromart) [inline cmd/mesh/testdata/operator/output/operator-init.yaml]: Also looks suspect...

**Comment by ostromart** [comment]
make update-goldens now works in the container build. 
Just do:
make update-goldens
make clean gen

**Comment by pitlv2109** [comment]
> make update-goldens now works in the container build.
> Just do:
> make update-goldens
> make clean gen

Running these commands didn't give me any new changes. Let me update this PR with just my file and the generated `pkg/vfs/assets.gen.go` file and see if the tests will pass

**Review by howardjohn — APPROVED** [review]

**Comment by pitlv2109** [comment]
I'll cherry-pick manually

**Pull request merged**